clash-prelude/src/Clash/Explicit/BlockRam.hs

```diff
@@ -729,12 +738,24 @@ prog2 = -- 0 := 4
 
 -}
 
--- | Create a blockRAM with space for @n@ elements
+-- | Create a block RAM with space for @n@ elements
 --
 -- * __NB__: Read value is delayed by 1 cycle
 -- * __NB__: Initial output value is /undefined/, reading it will throw an
 -- 'XException'
 --
+-- Additional helpful information:
```

Comment:
Or possibly just "See Also" so the same heading can refer to snippets of documentation _and_ related functions etc.

Reply:
I made it a heading, and made it "See also:", no title case, ending in a colon.

Reply:
_No_ title case? Why not?

Reply:
I don't see us use title case in headings? I checked `Clash.Explicit.BlockRam` as it was the file I was editing and `Clash.Tutorial` as I expected it to have a lot of headings. Neither use title case. I just followed what was already there.

I can also give a specific reason: I already think the "See also:" heading is pretty heavy, adding title case only increases its weight.

Reply:
> I don't see us use title case in headings?

If it's more consistent for now then by all means keep it that way. If we consistently use sentence case for headings then we could also just adopt that as convention and remove that style question permanently.

I mentioned it some time recently, but perhaps it would be a good idea to have a style guide for documentation. I think it would be a good effort to make in terms of improving the quality of documentation if we set some rules for how it should be written.